### PR TITLE
Fix textfield state.

### PR DESCRIPTION
**Pull request by dmytrokhl**
## Description of changes in release / Impact of release:
Fix textfield state when user put more data then limited by mask.

## Screenshots / gif (if customer facing):
(insert image here)

## Value being added to the product or risk being removed by this release
(insert text here)

## Changes to documentation (put link to pull request if some EXISTING docs page needs changes) 
- [ -] My change requires a change to the documentation.
(insert link here)
- [ -] I have updated the documentation accordingly.
(insert link here)

## Backout procedure

Deploy the previous release.

### CODE REVIEW

File: framework/Sources/VGSFramework/UIElements/Text Field/Mask/MaskedTextField.swift [hunk 1]
```diff
@@ -53,12 +53,15 @@ open class MaskedTextField: UITextField {
         set {
             super.text = newValue
             self.formatText()
+            self.onEditingChange?()
```
Comment (miraving) [inline framework/Sources/VGSFramework/UIElements/Text Field/Mask/MaskedTextField.swift]: Noooo. Named incorrect. This is the JS style of naming. Say no for `on...` or `click...` )) 
I guess will be good `self.editingChange?()`

Comment (miraving) [reply framework/Sources/VGSFramework/UIElements/Text Field/Mask/MaskedTextField.swift]: And need to add `[weak self] in` Or maybe I'm wrong?

File: framework/Sources/VGSFramework/UIElements/Text Field/VGSTextField.swift [hunk 2]
```diff
@@ -138,8 +138,12 @@ public class VGSTextField: UIView {
...
         NSLayoutConstraint.activate(verticalConstraint)
         
-        //delegate
-        textField.addSomeTarget(self, action: #selector(textFieldDidChange), for: .editingChanged)
+        textField.onEditingChange = { [weak self] in
+            if let strongSelf = self {
+                strongSelf.vgsCollector?.updateStatus(for: strongSelf)
+            }
+        }
+         //delegates
```
Comment (miraving) [inline framework/Sources/VGSFramework/UIElements/Text Field/VGSTextField.swift]:
````swift
textField.onEditingChange = { [weak self] in
    self?.vgsCollector?.updateStatus(for: strongSelf)
}
````
- more beautiful format

Comment (dmytrokhl) [reply framework/Sources/VGSFramework/UIElements/Text Field/VGSTextField.swift]: **updateStatus**: takes required instance, not **self?**

**Pull request merged**